### DISCUSSION TITLE
I selected Windows 11 taskbar and I can no longer open the settings to turn it back

### DISCUSSION THREAD

**Discussion by highlanderc** [valinet/ExplorerPatcher#788] [General]
Was having an issue and in trying to fix it, i can no longer access the properties for explorer patcher. I selected Windows 11 taskbar and I can no longer open the settings to turn it back, it always shows taskbar settings for windows 11 and never explorer patcher settings.

Any other way to access this menu?

Thanks

**Comment by Andre-10** [comment]
Start Task Manager, and select File / New task. type in (copy and paste):

`rundll32.exe C:\Windows\dxgi.dll,ZZGUI`

that's it

**Reply by highlanderc** [reply]
THANK YOU!
